foundations/foundation-vertx/src/test/java/io/servicecomb/foundation/vertx/http/TestFileUploadPart.java

```diff
@@ -0,0 +1,138 @@
+/*

```

Comment:
Need to update the license header.

Reply:
sorry, did not update code template at home

Reply:
done